src/domUpdates.js

```diff
@@ -13,8 +14,81 @@ const domUpdates = {
 
   populateDestinationDropDown: (destination) => {
     $('#select-destination').append(`<option value="${destination.destination}">${destination.destination}</option>`)
-  }
+  },
 
+  //If cannot be tested yet. Need to do a POST
+  populateTravelerInfo: (traveler, travelerID, tripData, destinationData) => {
+    if (traveler.findCurrentTrips(travelerID, tripData)) {
+      let tripDestination = destinationData.find(destination => traveler.findCurrentTrips(travelerID, tripData).destinationID === destination.id)
+      $('.display-username').text(`Welcome ${traveler.name}, enjoy ${tripDestination.destination}!`)
+    } else {
+      $('.display-username').text(`Welcome ${traveler.name}!`)
+    }
+    $('.display-username').text(`Welcome ${traveler.name}!`)
+    $('.display-money').text(`You Spent $${traveler.calculateUserSpent(travelerID, tripData, destinationData)} This Year`)
+  },
+
+  populateUpcomingTrips: (traveler, travelerID, tripData, destinationData) => {
+    let userTrips = traveler.findUserUpcomingTrips(travelerID, tripData)
+    userTrips.forEach(trip => {
+      let foundDestination = destinationData.find(destination => destination.id === trip.destinationID)
+      $('.upcoming-trip-cards').append(`
+        <div class="trip-card">
+          <img src=${foundDestination.image} class="card-background" alt="${foundDestination.destination}" />
```

Comment:
This is the one that you are trying to get to display the image as background that you were asking about? Does the alt generate as it should, at least? Or do neither work?

Reply:
Maybe it's because you are using img and not background image?

Reply:
Yea so I tried using the background image with css but cant figure out how to target the card being created at that time resulting in many images or over writing the background image of all previously generated cards.